src/NServiceBus.Core/App_Packages/Particular.Licensing/FindActiveLicense/ActiveLicense.cs

```diff
@@ -20,15 +20,22 @@ public static ActiveLicenseFindResult Find(string applicationName, params Licens
             var licenseSourceResultToUse = LicenseSourceResult.DetermineBestLicenseSourceResult(results.ToArray());
             if (licenseSourceResultToUse != null)
             {
-                activeLicense.Report.Add($"Selected active license from {licenseSourceResultToUse.Location}");
+                var selectedLicenseReportItem = $"Selected active license from {licenseSourceResultToUse.Location}";
+                activeLicense.Report.Add(selectedLicenseReportItem);
```

Comment:
>I know this is nitpicking, but now that we have spent so much time and effort on coming up with proper ways to optimize the logging, it feels weird to live with such a compromise :

Hmm, I guess I don't yet see how it matters. You're going to get two separate log entries, one a DEBUG, and one an INFO.

I suppose we could do something like checking the log level and then only doing one or the other, since the DEBUG entry would have everything the INFO one does.